docs/docs/PythonUserGuide/run-without-pip.md

```diff
@@ -1,101 +1,70 @@
-## **Precondition**
-First of all, you need to obtain the BigDL libs. Refer to [Install from pre built](../ScalaUserGuide/install-pre-built.md) or [Install from source code](../ScalaUserGuide/install-build-src.md) for more details
```

Comment:
Keep this? as pip install would not always meet user's requirement. i.e run in cluster mode (at least pip doesn't support cluster mode for now)